install_config/cluster_metrics.adoc

```diff
@@ -304,13 +304,15 @@ appended to the prefix starting from 1.
 |The persistent volume claim size for each of the Cassandra nodes.
 
 |`openshift_metrics_cassandra_pvc_storage_class_name`
-|If you want to explicitly set the storage class, you must not set
-`openshift_metrics_cassandra_storage_type` to `emptydir` or `dynamic`.
+|Specify the storage class to use. If you want to explicitly set the storage class, 
+use `openshift_logging_elasticsearch_storage_type=pv`. This parameter does not work 
```

Comment:
@mburke5678  Do you ream openshift_metrics_cassandra_storage_type=pv here?

Reply:
@anpingli 
Yes. openshift_metrics_cassandra_storage_type=pv  Thank you!